dojo/benchmark/views.py

```diff
@@ -28,71 +39,79 @@ def add_benchmark(queryset, product):
 
     try:
         Benchmark_Product.objects.bulk_create(requirements)
-    except:
+    except BaseException:
```

Comment:
```suggestion
    except Exception:
```

Reply:
Changes apply in commit https://github.com/DefectDojo/django-DefectDojo/pull/8321/commits/5777783dc6f53ae0172a58eeb289b20c72f75997